src/2_resources.js

```diff
@@ -129,8 +129,7 @@ resources.linkClick = {
 		"link_url": validator(true, validationTypes.STRING)
 	},
 	params: {
-		"click": validator(true, validationTypes.STRING),
-		"identity": validator(true, validationTypes.STRING)
```

Comment:
The first bool arg to `validator` decides whether the parameter is required. Why not instead of removing do we not just set it to `false`?

Reply:
Same with the other ones below. I am more actually curious, not actually sure if it's fair to just make it optional with validation. It just sounds better to me in theory.